I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/labels/common1.rpy", line 165, in script
    scene bg stage with longFade
TypeError: Fade() takes at least 3 arguments (3 given)

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/labels/common1.rpy", line 165, in script
    scene bg stage with longFade
  File "C:\Users\John Dough\Documents\renpy-7.3.5-sdk.7z\renpy-7.3.5-sdk\renpy\ast.py", line 1389, in execute
    renpy.exports.with_statement(trans, paired)
  File "C:\Users\John Dough\Documents\renpy-7.3.5-sdk.7z\renpy-7.3.5-sdk\renpy\exports.py", line 1601, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "C:\Users\John Dough\Documents\renpy-7.3.5-sdk.7z\renpy-7.3.5-sdk\renpy\display\core.py", line 2251, in do_with
    clear=clear)
  File "C:\Users\John Dough\Documents\renpy-7.3.5-sdk.7z\renpy-7.3.5-sdk\renpy\display\core.py", line 2702, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, **kwargs)
  File "C:\Users\John Dough\Documents\renpy-7.3.5-sdk.7z\renpy-7.3.5-sdk\renpy\display\core.py", line 3047, in interact_core
    new_widget=layers_root)
  File "C:\Users\John Dough\Documents\renpy-7.3.5-sdk.7z\renpy-7.3.5-sdk\renpy\curry.py", line 43, in __call__
    **dict(self.kwargs.items() + kwargs.items()))
TypeError: Fade() takes at least 3 arguments (3 given)

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Ganymead 1.0
Tue Apr 21 19:38:51 2020
